data/json/items/comestibles.json

```diff
@@ -17295,6 +17321,82 @@
     },
     {
         "type" : "COMESTIBLE",
+        "id" : "sandwich_dudeluxe",
+        "name" : "dudeluxe sandwich",
+        "name_plural" : "dudeluxe sandwiches",
+        "weight" : 340,
+        "color" : "brown",
+        "addiction_type" : "none",
+        "spoils_in" : 32,
+        "stim" : 1,
+        "container" : "wrapper",
+        "comestible_type" : "FOOD",
+        "symbol" : "%",
+        "quench" : 1,
+        "healthy" : 1,
+        "addiction_potential" : 0,
+        "nutrition" : 38,
+        "description" : "A sandwich of human flesh, vegetables, and cheese with condiments.  Feast upon the souls of your enemies and tasty garden greens!",
+        "price" : 1200,
+        "material" : ["hflesh", "veggy", "wheat", "milk"],
+        "tool" : "null",
+        "volume" : 1,
+        "charges" : 4,
+        "fun" : 12
+    },
+    {
+        "type" : "COMESTIBLE",
+        "id" : "sandwich_deluxe",
+        "name" : "deluxe sandwich",
+        "name_plural" : "deluxe sandwiches",
+        "weight" : 340,
+        "color" : "brown",
+        "addiction_type" : "none",
+        "spoils_in" : 32,
+        "stim" : 1,
+        "container" : "wrapper",
+        "comestible_type" : "FOOD",
+        "symbol" : "%",
+        "quench" : 1,
+        "healthy" : 1,
+        "addiction_potential" : 0,
+        "nutrition" : 38,
+        "description" : "A sandwich of meat, vegetables, and cheese with condiments. Tasty and nutritious!",
+        "price" : 1200,
+        "material" : ["flesh", "veggy", "wheat", "milk"],
+        "tool" : "null",
+        "volume" : 1,
+        "charges" : 4,
+        "fun" : 12
+    },
+    {
+        "type" : "COMESTIBLE",
+        "id" : "sandwich_cucumber",
+        "name" : "cucmber sandwich",
+        "name_plural" : "cucumber sandwiches",
+        "weight" : 187,
+        "color" : "light_green",
+        "addiction_type" : "none",
+        "spoils_in" : 36,
+        "stim" : 0,
+        "container" : "wrapper",
+        "comestible_type" : "FOOD",
+        "symbol" : "%",
+        "quench" : 2,
+        "healthy" : 1,
+        "addiction_potential" : 0,
+        "nutrition" : 26,
```

Comment:
Checked math on nutrition. Bread: 2x20 Cucumber: 11 Total: 51
                                            Nutrition: 26   Charges: 2    Total: 52

It had not occurred to me to check the nutrition values. Thank you for pointing it out.